README.md

```diff
@@ -9,6 +9,11 @@
 
 Manages staging directory, along with download/extraction of compressed files.
 
+**Use of this module is deprecated.**
+New features are unlikely to be added.
+Please consider using [puppet-archive](https://github.com/voxpupuli/puppet-archive)
```

Comment:
This link should probably be updated to point to the migration guide. https://github.com/voxpupuli/puppet-archive/issues/266